src/effects/onSuccessPagination/index.js

```diff
@@ -0,0 +1,27 @@
+import { mergeState } from '../../configuration';
+
+// TODO: Add support and validations for multi target actions
+
+function onSuccessPagination(selector = action => action.payload) {
+  return (state, action) =>
+    (selector(action).list
+      ? mergeState(state, {
+        [`${action.target}Loading`]: false,
+        [`${action.target}Error`]: null,
+        [`${action.target}`]:
+            Number(selector(action).meta.currentPage) === 1
+              ? selector(action).list
+              : state[action.target].concat(selector(action).list),
+        [`${action.target}TotalPages`]: Number(selector(action).meta.totalPages),
```

Comment:
I think you need to modify `completeState` as well, in order to add all these new targets